File: snippets/php/array/add/add.php
<?php
$arraay = [3, 2];
$item = 1;
var_export($arraay);  // array (0 => 3, 1 => 2, )
$arraay[] = $item;  // ≡
var_export($arraay);  // array (0 => 3, 1 => 2, 2 => 1, )
?>

File: snippets/rs/array/add/add.rs
fn main()
{
    let mut array = vec![3, 2];
    let item = 1;
    println!("{array:?}");  // [3, 2]
    array.push(item);  // ≡
    println!("{array:?}");  // [3, 2, 1]
}